src/main/java/edu/kis/powp/jobs2d/drivers/InkUsageDriver.java

```diff
@@ -0,0 +1,60 @@
+package edu.kis.powp.jobs2d.drivers;
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.logging.Logger;
+
+public class InkUsageDriver implements Job2dDriver {
+
+    private int x0, y0;
+    private double totalUsage;
+    Logger logger = Logger.getLogger("global");
+    private DrawPanelController drawController;
+    private ILine line;
+
+
+
+    public InkUsageDriver(ILine line, DrawPanelController drawController){
```

Comment:
Implementing another drawing driver is not necessary.  Use decorator pattern approach.
Too many responsibilities in this class. Even the usage aspect should be extracted (e.g. separate interface)